src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper.ts

```diff
@@ -0,0 +1,931 @@
+    assert.assert(params.amount <= unassignedTokens, 'Amount is greater than unassigned tokens');
+
+    const canTransferResult = await (await this.securityTokenContract()).canTransfer.callAsync(
+      await this.getTreasuryWallet(),
+      numberToBigNumber(params.amount),
+      '0x00',
+    );
+    assert.assert(canTransferResult[0] === TRANSFER_SUCCESS, 'Transfer failed');
+
+    return (await this.contract).sendToTreasury.sendTransactionAsync(
+      numberToBigNumber(params.amount),
+      params.txData,
+      params.safetyFactor,
+    );
+  };
+
+  /**
+   * Returns the treasury wallet address
+   */
+  public getTreasuryWallet = async () => {
+    return (await this.contract).getTreasuryWallet.callAsync();
+  };
+
+  /**
+   * Pushes available tokens to the beneficiary's address
+   */
+  public pushAvailableTokens = async (params: PushAvailableTokensParams) => {
+    assert.assert(await this.isCallerAllowed(params.txData, Perm.Operator), 'Caller is not allowed');
+
+    return (await this.contract).pushAvailableTokens.sendTransactionAsync(
+      params.beneficiary,
+      params.txData,
+      params.safetyFactor,
+    );
+  };
+
+  /**
+   * Used to withdraw available tokens by beneficiary
+   */
+  public pullAvailableTokens = async (params: TxParams) => {
+    assert.assert(!(await this.paused()), 'Contract currently paused');
+    return (await this.contract).pullAvailableTokens.sendTransactionAsync(params.txData, params.safetyFactor);
+  };
+
+  /**
+   * Adds template that can be used for creating schedule
+   */
+  public addTemplate = async (params: AddTemplateParams) => {
+    assert.assert(await this.isCallerAllowed(params.txData, Perm.Admin), 'Caller is not allowed');
+    assert.assert(params.name !== '', 'Invalid name');
+    assert.assert(!(await this.getAllTemplateNames()).includes(params.name), 'Template name already exists');
+    await this.validateTemplate(params.numberOfTokens, params.duration, params.frequency);
+    return (await this.contract).addTemplate.sendTransactionAsync(
+      stringToBytes32(params.name),
+      valueToWei(numberToBigNumber(params.numberOfTokens), FULL_DECIMALS),
+      numberToBigNumber(params.duration),
+      numberToBigNumber(params.frequency),
+      params.txData,
+      params.safetyFactor,
+    );
+  };
+
+  /**
+   * Removes template with a given name
+   */
+  public removeTemplate = async (params: RemoveTemplateParams) => {
+    assert.assert(await this.isCallerAllowed(params.txData, Perm.Admin), 'Caller is not allowed');
+    assert.assert(params.name !== '', 'Invalid name');
+    assert.assert((await this.getAllTemplateNames()).includes(params.name), 'Template not found');
+    // TODO 3.1: require(templateToUsers[_name].length == 0, "Template is used");
+    return (await this.contract).removeTemplate.sendTransactionAsync(params.name, params.txData, params.safetyFactor);
+  };
+
+  /**
+   * Returns count of the templates those can be used for creating schedule
```

Comment:
Change it to

`Returns the amount of templates that can be used for creating schedules`